tests/test_scylla_docker_cluster.py

```diff
@@ -1,4 +1,46 @@
+import re
+
 class TestScyllaDockerCluster:
+    @staticmethod
+    def parse_nodetool_status(lines):
+        """parse output of nodetool status
+
+        Nodetool status output:
+        Datacenter: eu-west
+        ===================
+        Status=Up/Down
+        |/ State=Normal/Leaving/Joining/Moving
+        --  Address      Load       Tokens       Owns    Host ID                               Rack
+        UN  10.0.15.114  36.53 GB   256          ?       98429fc3-1e89-4029-ac1c-325179752142  1a
+        UN  10.0.126.57  88.17 GB   256          ?       aea7e0f2-c2c3-4dc6-8ffd-8eda27f4ab8e  1a
+        UN  10.0.74.155  90.62 GB   256          ?       f2df2267-b8d1-4a1b-a5d8-a6c57a289f44  1a
+        UN  10.0.65.254  101.39 GB  256          ?       20eca592-3eda-478b-b9c8-03266879b8ba  1a
+
+        Parsed result:
+        [
+            {"status": "UN", "address": "10.0.15.114", size: "36.53", dimension: KB|MB|GB},
+            {"status": "UN", "address": "10.0.126.57", size: "88.17", dimension: KB|MB|GB},
+            {"status": "UN", "address": "10.0.74.155", size: "90.62", dimension: KB|MB|GB},
+            {"status": "UN", "address": "10.0.65.254", size: "101.39", dimension: KB|MB|GB}
+
+        ]
+        """
+        keys = ["status", "address", "size", "dimension"]
+        nodes_statuses = []
+        line_re = re.compile(
```

Comment:
1. The IP address will be only `IPv4` address?
2. some fields not parsed, why?

Reply:
for this simple test case, we could assume that the fields we need are here, and that most likely the cluster will use IPv4... if we get to work with IPv6 we could update this test to reflect it